stable/influxdb/templates/service-headless.yaml

```diff
@@ -0,0 +1,50 @@
+apiversion: v1
+kind: Service
+metadata:
+  name: {{ template "influxdb.fullname" . }}-headless
+labels:
+  app: {{ template "influxdb.fullname" . }}
+  chart: {{ template "influxdb.chart" . }}
+  release: {{ .Release.Name }}
+  heritage: {{ .Release.Service }}
```

Comment:
We've updated labels since `4.0.0` you can use:

```
metadata:
  name: {{ include "influxdb.fullname" . }}
  labels:
    {{- include "influxdb.labels" . | nindent 4 }}
```